specification/healthdataaiservices/HealthDataAIServices.DeidServices/models.tsp

```diff
@@ -196,182 +22,13 @@ union OperationType {
-  etag?: eTag;
-}
-
-@doc("List of PHI Entities.")
-union PhiCategory {
-  @doc("Unknown PHI Type.")
-  Unknown: "Unknown",
-
-  @doc("Account Number.")
-  Account: "Account",
-
-  @doc("Age.")
-  Age: "Age",
-
-  @doc("Biological Identifier, such as a fingerprint or retinal scan.")
-  BioID: "BioID",
-
-  @doc("City.")
-  City: "City",
-
-  @doc("Country or Region.")
-  CountryOrRegion: "CountryOrRegion",
-
-  @doc("Date.")
-  Date: "Date",
-
-  @doc("Device ID or serial numbers.")
-  Device: "Device",
-
-  @doc("Doctor's Name.")
-  Doctor: "Doctor",
-
-  @doc("Email Addresses.")
-  Email: "Email",
-
-  @doc("Fax Number.")
-  Fax: "Fax",
-
-  @doc("Health Plan ID Numbers.")
-  HealthPlan: "HealthPlan",
-
-  @doc("Hospital Name.")
-  Hospital: "Hospital",
-
-  @doc("Id Number, eg. passport number.")
-  IDNum: "IDNum",
-
-  @doc("IP Address.")
-  IPAddress: "IPAddress",
-
-  @doc("License, eg. Driver's license or medical license.")
-  License: "License",
-
-  @doc("Location Other, eg. Golden Gate Park.")
-  LocationOther: "LocationOther",
-
-  @doc("Medical Record Number.")
-  MedicalRecord: "MedicalRecord",
-
-  @doc("Organization, eg. Microsoft.")
-  Organization: "Organization",
-
-  @doc("Patient Name.")
-  Patient: "Patient",
-
-  @doc("Phone Number.")
-  Phone: "Phone",
-
-  @doc("Profession.")
-  Profession: "Profession",
-
-  @doc("Social Security Number.")
-  SocialSecurity: "SocialSecurity",
-
-  @doc("State.")
-  State: "State",
-
-  @doc("Street.")
-  Street: "Street",
-
-  @doc("Web URL.")
-  Url: "Url",
-
-  @doc("Usernames, eg. a social media handle.")
-  Username: "Username",
-
-  @doc("Vehicle IDs, eg. license plate or VIN number.")
-  Vehicle: "Vehicle",
-
-  @doc("Zip Code.")
-  Zip: "Zip",
-
-  string,
-}
-
-@doc("PHI Entity tag in the input.")
-model PhiEntity {
-  @doc("PHI Category of the entity.")
-  category: PhiCategory;
-
-  @doc("Starting index of the location from within the input text.")
-  offset: StringIndex;
-
-  @doc("Length of the input text.")
-  length: StringIndex;
-
-  @doc("Text of the entity.")
-  text?: string;
-
-  @doc("Confidence score of the category match.")
-  confidenceScore?: float64;
-}
-
-@doc("Query parameters for list operation.")
-model PaginationByTokenQueryParameters {
-  ...MaxPageSizeQueryParameter;
-
-  @query
-  @doc("Token to continue a previous query.")
-  continuationToken?: string;
-}
-
-@doc("String index encoding model.")
-model StringIndex {
-  @doc("The offset or length of the substring in UTF-8 encoding")
-  utf8: int32;
-
-  @doc("""
-    The offset or length of the substring in UTF-16 encoding.
-    
-    Primary encoding used by .NET, Java, and JavaScript.
-    """)
-  utf16: int32;
 
-  @doc("""
-    The offset or length of the substring in CodePoint encoding.
-    
-    Primary encoding used by Python.
-    """)
-  codePoint: int32;
+  @doc("Locale in which the output surrogates are written.")
+  surrogateLocale?: string = "en-US";
```

Comment:
Hmmmm... This is a good point. My plan was to return a list of supported locales in the `BadRequest` response message if they send something we don't support, in addition to documentation.

We could try utilizing an enum here, I wanted to avoid it as then anytime we want to release new language support we will have to create a new version and go through SDK release process.

Reply:
The documentation + error message solution sounds good to me.